washer-start/functions/index.js

```diff
@@ -69,16 +69,32 @@ const app = smarthome({
 app.onSync((body) => {
   // TODO: Implement SYNC response
   return {
-    'requestId': 'ff36a3cc-ec34-11e6-b1a0-64510650abcf',
-    'payload': {
-      'agentUserId': '123',
-      'devices': [],
+    requestId: 'ff36a3cc-ec34-11e6-b1a0-64510650abcf',
+    payload: {
+      agentUserId: '123',
+      devices: [],
     },
 
   };
 //  return {};
 });
 
+const queryFirebase = (deviceId) => firebaseRef.child(deviceId).once('value')
+  .then((snapshot) => {
+    const snapshotVal = snapshot.val();
+    return {
+      on: snapshotVal.OnOff.on,
```

Comment:
It's fine to return all of the parameters in `queryFirebase` as a response to a `QUERY` intent. The Report State call does need to be more restrictive.

Reply:
I understand.